Data/Base/Materials/BaseMaterials/TestBricks.ezMaterial

```diff
@@ -25,12 +25,36 @@ Constant
   Color %Value { f{0.3, 0.3, 0.3, 1.0} }
 }
 
+Constant

```

Comment:
Fix for the decal preview material not working correctly if a material with other settings was used before this.